api/director_service.go

```diff
@@ -262,6 +270,19 @@ func (a Api) addGUIDToExistingNetworks(networks Networks) (Networks, error) {
 	return networks, nil
 }
 
+func (a Api) addGUIDToExistingSubnet(network *Network, existingNetwork *Network) []Subnet {
+	for k, subnet := range network.Subnets {
+		for _, existingSubnet := range existingNetwork.Subnets {
+			if subnet.CIDR == existingSubnet.CIDR {
```

Comment:
Let me know if the comparison should be made on more fields. In theory it does not really matter since the other field would be updated.